dkc/core/tests/test_authorized_uploads.py

```diff
@@ -0,0 +1,179 @@
+from datetime import timedelta
+
+from django.utils import timezone
+import pytest
+
+from dkc.core.models import AuthorizedUpload
+
+
+@pytest.fixture
+def authorized_upload(admin_api_client, folder):
+    return admin_api_client.post('/api/v2/authorized_uploads', data={'folder': folder.id})
```

Comment:
I think this should be a Factory and operate at the model layer instead. A fixture is part of the surface under test, why go through the API?

Reply:
I had done this prior to making `signature` a model property, but now that it is, there's no need for this. I'll change it.